packages/wdio-utils/src/monad.ts

```diff
@@ -3,18 +3,33 @@ import logger from '@wdio/logger'
 
 import { commandCallStructure, overwriteElementCommands } from './utils'
 
-const SCOPE_TYPES = {
-    'browser': /* istanbul ignore next */ function Browser () {},
-    'element': /* istanbul ignore next */ function Element () {}
+const SCOPE_TYPES: Record<string, Function> = {
+    browser: /* istanbul ignore next */ function Browser () {},
+    element: /* istanbul ignore next */ function Element () {}
 }
 
-export default function WebDriver (options, modifier, propertiesObject = {}) {
+interface PropertiesObject {
```

Comment:
Is it possible to deal with `any`?

Reply:
I don't want to import any Puppeteer types just because I had to type it for something that we don't use here:

```js
        /**
         * assign propertiesObject to itself so the client can be recreated
         */
        // eslint-disable-next-line @typescript-eslint/no-unused-vars
        const { puppeteer, ...propertiesObjectWithoutPuppeteer } = propertiesObject
        propertiesObject['__propertiesObject__'] = { value: propertiesObjectWithoutPuppeteer }
```

Reply:
Makes sense